packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx

```diff
@@ -158,8 +158,9 @@ export class FileUploader {
 
   /**
    * errors - errors collection.
+   * Mutable as this can be set from form control too based on form validations.
    */
-  @State() errors: any = [];
+  @Prop({ mutable: true }) errors: any = [];
```

Comment:
Can you introduce props like State=‘error’|’warning’|’hint’ 
`Errortext, hinttext, warningtext` props similar to what we have in `input` , `select` `datepicker` etc.?

Reply:
Changed to `errorText` and `hintText` as suggested. As discussed over call, did not use state prop as it was not valid for file uploader.